packages/manager/modules/cloud-universe-components/src/space-meter/space-left/tooltip.html

```diff
@@ -1,9 +1,4 @@
-<cuc-space-meter-space-left
-    data-usage="$ctrl.usage"
-    data-direction="'column'"
-    data-strong="true">
-</cuc-space-meter-space-left>
-
+<span>{{ $ctrl.usage | cucSpaceMeterSpaceLeft }}</span>
```

Comment:
The behaviour is changing here. I'm not sure it shoud be part of this PR

Reply:
The removed code, calls the component from inside component. This makes it to go into loop and slows down freezes the UI. The change displays same information and looks the same as before.